### PR TITLE
point to 1.2 branch for velero and csi plugin on master

### PR DESCRIPTION
**Pull request by sseago**

### CODE REVIEW

**Review by shubham-pampattiwar — APPROVED** [review]

File: pkg/common/common.go [hunk 1]
```diff
@@ -23,12 +23,12 @@ const (
 
 // Images
 const (
-	VeleroImage          = "quay.io/konveyor/velero:latest"
+	VeleroImage          = "quay.io/konveyor/velero:oadp-1.2"
```
Comment (kaovilai) [inline pkg/common/common.go:26]: don't actually need these. IIRC we just read from ENV first.. only if ENV is empty/malformed do these hardcoded backups take effect.

Comment (kaovilai) [reply pkg/common/common.go:26]: def non issue.. just a note.

Comment (sseago) [reply pkg/common/common.go:26]: @kaovilai I was just following what we did for oadp-1.2 branch. In any case, if it's harmless, lets just leave it for now.

**Comment by kaovilai** [comment]
keeping builder:v1.19 for now to minimize changes. added GOBIN to path as a temp workaround

**Review by shubham-pampattiwar — APPROVED** [review]
/lgtm

**Review by kaovilai — APPROVED** [review]

**Pull request merged**